k8s.gcr.io/images/k8s-staging-build-image/images.yaml

```diff
@@ -9,7 +9,7 @@
     "sha256:248087165d8f4d901e27a799f10fe4b7b4458469191120385317ad32aa5e3382": ["buster-v1.10.0"]
     "sha256:2f37650255a457d0fc3a5c685ac1eecb5e0f16c63b58b82e52a585fff0aabff6": ["buster-v1.7.2"]
     "sha256:36652ef8e4dd6715de02e9b68e5c122ed8ee06c75f83f5c574b97301e794c3fb": ["buster-v1.4.0"]
-    "sha256:37cfe133a6ff3fc3aa4aa5ab9fda127861902940b8e8078fff7191c7a81be8d8": ["buster-v1.1.3","v2.1.3"]
+    "sha256:37cfe133a6ff3fc3aa4aa5ab9fda127861902940b8e8078fff7191c7a81be8d8": ["buster-v1.1.3", "v2.1.3"]
```

Comment:
`cip-mm` did it on its own for some reason. I don't know should we keep or drop this change.